core/src/replay_stage.rs

```diff
@@ -629,11 +629,19 @@ impl ReplayStage {
             // errors related to the slot being purged
             let slot = bank.slot();
             warn!("Fatal replay error in slot: {}, err: {:?}", slot, err);
-            datapoint_error!(
-                "replay-stage-mark_dead_slot",
-                ("error", format!("error: {:?}", err), String),
-                ("slot", slot, i64)
-            );
+            if err.is_severity_error() {
```

Comment:
Did you mean `is_severe_error`?

Reply:
I was trying to communicate the severity of the error: info, trace, error

Reply:
Doesn't make sense to me, but I won't block you

Reply:
I'd rather just remove `is_severity_error()` and inline the `matches!`.  There's no other use for `is_severity_error()` currently, so no need to build a new generic thing for something that is very specific

Reply:
sg, removed!